examples/puts3_config.yml

```diff
@@ -0,0 +1,72 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+MiNiFi Config Version: 3
+Flow Controller:
+  name: MiNiFi Flow
+Processors:
+- Properties:
+    Input Directory: /tmp/input
+  auto-terminated relationships list: []
+  class: org.apache.nifi.processors.standard.GetFile
+  id: bfd376a8-c845-453b-9f74-4cf6f3a95e74
+  name: GetFile
+  penalization period: 30 sec
+  run duration nanos: 0
+  scheduling period: 2 sec
+  scheduling strategy: TIMER_DRIVEN
+  yield period: 1 sec
+- Properties:
+    Access Key: test_access_key
+    Bucket: test_bucket
+    Secret Key: test_secret
+  auto-terminated relationships list:
+  - failure
+  class: org.apache.nifi.processors.standard.PutS3Object
+  id: 8c9e2512-e814-4b73-ac5c-f33753332128
+  name: PutS3Object
+  penalization period: 30 sec
+  run duration nanos: 0
+  scheduling period: 1 sec
+  scheduling strategy: TIMER_DRIVEN
+  yield period: 1 sec
+- Properties:
+    Directory: /tmp/output
+    Directory Permissions: '777'
+    Permissions: '777'
+  auto-terminated relationships list:
+  - success
+  - failure
+  class: org.apache.nifi.processors.standard.PutFile
```

Comment:
I find this layout confusing. Consider following the format of `CONFIGURE.md`:
```
    - name: GetFile
      id: 471deef6-2a6e-4a7d-912a-81cc17e3a206
      class: org.apache.nifi.processors.standard.GetFile
      max concurrent tasks: 1
      scheduling strategy: TIMER_DRIVEN
      scheduling period: 1 sec
      penalization period: 30 sec
      yield period: 1 sec
      run duration nanos: 0
      auto-terminated relationships list:
      Properties:
          Input Directory: /tmp/getfile
          Keep Source File: true
```

Also, I would exclude unnecessary settings from the example config, like "scheduling period" in case of EVENT_DRIVEN strategy. Or yield period, unless it has some significance.

Reply:
As an example, this is the config I was using to test ConsumeJournald: https://pastebin.com/svRPwyMh

Reply:
Changed the files have a consistent layout in 369f1085d136a749731b067dad19c73cded52532. Also removed the optional properties that were just making noise. 

I really liked your example, it's simple, on the point for anyone who wants to use ConsumeJournald processors, so I added that to the list of examples in 5583561941397e5605c339cb7e15148ac411f286